src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
```

Comment:
Good job on the consistent following of pascal/camel casing for class, method and variable names!